plugins/woocommerce/client/admin/client/wp-admin-scripts/fulfillments/index.tsx

```diff
@@ -43,8 +46,14 @@ function FulfillmentsController() {
 					setIsOpen( true );
 				}
 			}
-		} );
-	}, [] ); // eslint-disable-line react-hooks/exhaustive-deps
+		};
+
+		document.body.addEventListener( 'click', handleClick );
+
+		return () => {
+			document.body.removeEventListener( 'click', handleClick );
```

Comment:
`selectOrderRow` depends on `deselectOrderRow`... I wonder if you have experienced multiple registrations of the `click` event without removing previous registratations. I'm concerned the app could get more registrations of the event than deregistrations of it.

Reply:
The page is not a React page, it's a PHP rendered WP_List_Table, and every pagination change re-renders the full page, the event deregistration possibly never runs. The event registration is listening for click events on the body element, registering once per page load (not every render of the fulfillments drawer, it's running on the entry point of the script, which isn't bound to any react component), and checks if there are any `.fulfillments-trigger` element in the ascendants or descendants of the clicked element, then runs the code according to it. How can it register more than it's deregistered?